projects/angular-calendar/src/date-adapters/dayjs/index.ts

```diff
@@ -0,0 +1,136 @@
+import { DateAdapter } from '../date-adapter';
+import * as dayjs from 'dayjs';
+import isoWeek from 'dayjs/plugin/isoWeek';
+import minMax from 'dayjs/plugin/minMax';
+
+dayjs.extend(minMax);
+dayjs.extend(isoWeek);
+
+export function adapterFactory(): DateAdapter {
```

Comment:
This is I think the perfect file for heavy unit tests

Reply:
+1